internal/config/config.go

```diff
@@ -130,13 +131,31 @@ func (conf *Config) HasConfiguredOrganization(slug string) bool {
 	return ok
 }
 
-func (conf *Config) AddPreferredPipeline(pipelines []string) error {
-	conf.localConfig.Set("preferences.pipelines", pipelines)
-	return conf.localConfig.WriteConfig()
+func (conf *Config) PreferredPipelines() []pipeline.Pipeline {
```

Comment:
I changed these to take in the pipeline objects since I'd like those to be the things we pass around